zvariant/src/type.rs

```diff
@@ -675,6 +675,16 @@ impl_type! {
     }
 }
 
+#[cfg(feature = "chrono")]
+impl_type! {
+    chrono::Month => &str {
```

Comment:
I really appreciate you splitting the commits. However, I believe addition of `Type` for chrono types is one logical change, so I'd squash the commits and update the commit log. However, it's fine like this too, if you disagree.

Reply:
Squashing them makes sense to me 👍🏼